src/content/models.py

```diff
@@ -164,6 +164,12 @@ class Meta:
         null=True,
     )
 
+    confirmation_page_needed_at = models.DateTimeField(
+        default=None, null=True, blank=True
+    )
+
+    page_update_notified_at = models.DateTimeField(default=None, null=True, blank=True)
```

Comment:
similarly perhaps this might be
```suggestion
    archive_notification_sent_at = models.DateTimeField(default=None, null=True, blank=True)
```